### DISCUSSION TITLE
[Bug report] Terminal Theme or general - Accessibility Issue on Navigation

### DISCUSSION THREAD

**Discussion by tigexplorer** [GetPublii/Publii#1761] [Bug Report]
### Operating system

Linux ubuntu 24

### Publii version

46

### Issue type

Free theme

### Bug description

The Terminal Theme 2.0.0.0 does not comply with WCAG Accessibility Standards.

Multiple elements with "navigation" role do not have unique labels.
Each element with "navigation" role must have a unique label that describes its purpose.

This is the element location:

```<nav
    class="navbar js-navbar">
```

The IBM Equal Access Accessibility Checker says:

For each element with a "nav" role, either set the value of the aria-labelledby attribute to the id value of any visible text that appropriately and uniquely describes that navigation region
Or, use the aria-label attribute to add an invisible, unique, non-empty label to an element with the "nav" role

For example:

```
<h2 id="weather">Weather portlet</h2>
<nav aria-labelledby="weather">
	...
</nav>
    
<nav aria-label="Stock ticker portlet">
		...
</nav>

```

See also:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/navigation_role

**Comment by bobmitro** [comment]
Insignificant, most theme use one nav elements.
